src/wrapper.rs

```diff
@@ -0,0 +1,140 @@
+use std::{env, process::Command};
+
+use failure::format_err;
+
+pub(crate) const COMPILER_BUILTINS_RLIB_PATH_MARKER: &str =
+    "@CARGO_CALL_STACK:compiler_builtins_rlib_path@";
+pub(crate) const COMPILER_BUILTINS_LL_PATH_MARKER: &str =
+    "@CARGO_CALL_STACK:compiler_builtins_ll_path@";
+
+pub(crate) fn wrapper() -> Result<i32, failure::Error> {
+    let mut args = env::args().skip(1);
+    let rustc_path = args.next().unwrap();
+    let mut rustc = Command::new(&rustc_path);
+
+    let rustc_args = args.collect::<Vec<_>>();
+    let args = RustcArgs::parse(&mut rustc_args.iter().map(|s| &**s))?;
+
+    for ext in &args.extern_crates {
+        match (&*ext.crate_name, &ext.path) {
+            ("compiler_builtins", Some(path)) => {
+                eprintln!("{}{}", COMPILER_BUILTINS_RLIB_PATH_MARKER, path);
```

Comment:
(just curious if this will get printed several times since the rustc wrapper is invoked several times? I don't think it's a problem though)

Reply:
Yeah, the path should always be the same though (I can add an assertion)